src/modules/Builder/components/PreviewPhrasePopup/PreviewPhrasePopup.styles.ts

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@mui/material';
+
+import { theme, variables } from 'shared/styles';
+
+export const StyledImage = styled('img')({
+  borderRadius: variables.borderRadius.md,
+  flexShrink: 0,
+  height: 56,
+  objectFit: 'cover',
+  overflow: 'hidden',
+  width: 56,
```

Comment:
Just to stick with our existing dimension units convention (for what it's worth):
```suggestion
  height: '5.6rem',
  objectFit: 'cover',
  overflow: 'hidden',
  width: '5.6rem',
```

Reply:
[Updated](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1958/commits/3d830db0f977aae4da92e28eff5e8f8a0ae065b8)!